test/missing.jl

```diff
@@ -95,6 +95,7 @@ end
     for f in bit_operators
         @test ismissing(f(missing, missing))
     end
+    @test ismissing(~missing)
```

Comment:
It would make sense to add above it with unary operators (including `!`) instead.